### DISCUSSION TITLE
For those who are going to push to repo.

### DISCUSSION THREAD

**Discussion by realguystuff** [The-Lively-Developers-Community/Lively-Game#20] [Announcements]
Please note that COMMITS MUST HAVE VALID SIGNATURES. This is to prevent attacks (thanks GitHub)!

**Comment by realguystuff** [comment]
Vote up if all commits being pushed to the repo must be a PR so that we can check your code. Vote down if you do not want to.

**Comment by KendallDoesCoding** [comment]
> Please note that COMMITS MUST HAVE VALID SIGNATURES. This is to prevent attacks (thanks GitHub)!

If you use GIT, then it can't have verified signatures, so bad idea. Most people commit in Git these days, including me.

**Comment by realguystuff** [comment]
https://stackoverflow.com/questions/76246788/why-does-git-say-that-i-bypassed-the-rule-for-commits-must-have-valid-signature
![image](https://github.com/The-Lively-Developers-Community/Lively-Game/assets/101049350/78f55ef8-46ff-4404-859c-575386dd4481)
